docs/guides/partial-encryption.md

```
@@ -199,6 +210,103 @@ git commit -m "Migrate to partial encryption"
+secrets_only = true
+secrets_dir = "secrets/production/"
+pattern = ".env*"          # optional glob, default ".env*"
+```
+
+`pattern` is a standard glob applied inside `secrets_dir`. Only files matching it
+are encrypted/decrypted; everything else in the directory is left untouched.
+
+> **Note — non-recursive by default.** `pattern` is matched with `Path.glob`, which
+> does **not** descend into subdirectories unless the pattern itself contains `**`.
+> Use `**/.env*` if your secrets live in nested folders.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)